contracts/mocks/MockStrategy.sol

```diff
@@ -1,54 +1,63 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.9;
 
-// import {IVaultStrategy} from "../interfaces/IVaultStrategy.sol";
+import {IStrategy} from "../interfaces/IStrategy.sol";
+import {IERC20Detailed} from "../interfaces/IERC20Detailed.sol";
 
-contract MockStrategy {
-  uint public mockedListingId;
-  uint public mockedSize;
-  uint public mockedMinPremium;
+contract MockStrategy is IStrategy {
+  IERC20Detailed public immutable collateral;
+  IERC20Detailed public immutable premium;
```

Comment:
The new MockStrategy here should comply the new interface for us to properly test the vault lifecycle without having a real strategy.